operations/src/main/java/com/nextdoor/bender/operation/json/array/ArraySplitOperation.java

```diff
@@ -89,4 +101,11 @@ public ArraySplitOperation(String path) {
       return output;
     }
   }
+
+  private void addFieldsToNewJsonObject(InternalEvent internalEvent, JsonObject newObject) {
+    for (String field : fieldsToKeep) {
```

Comment:
Nit: use `this.fieldsToKeep`, also this is small enough that I don't think it needs a function.

Reply:
Sure will move back in the main logic. Using 'this.' just emphasizes the class's instance variable is used but I don't mind changing it.